CIP-0118/README.md

```diff
@@ -0,0 +1,1041 @@
+as well as some kind of computed conditions for updating datums.
+It also proposes that intent-processing be done by the node.
+
+We think that we can eventually move in the direction of adding more of the
+intent-settlement features from the proposal (e.g. conditions on values in inputs),
+but the major point of disagreement is about intent processing.
+As previously discussed, this proposal only focuses on intent settlement, not intent processing.
+We now briefly discuss options of how to possibly proceed in this direction.
+
+
+### Validation zones for settling other types of intents
+
+This CIP lays the foundation for specification and resolution of intents on Cardano.
+It does so by establishing the concept of a validation zone - a list of transactions that
+can be validated as a whole, but it is impossible to check if the individual transactions
+are valid outside a zone. We conjecture that infrastructure we propose can be used for processing
+other types of underspecified transactions, and therefore more sophisticated
+intents. A proof of this requires a
+fleshed-out design for the expanded intent-processing system, which is outside the
+scope of this CIP. However, possibilities include :
+
+- Intent transactions that allow output references to be removed and replaced with their corresponding
+output data, while intent-settling transactions provide the matching output references
+for those outputs to fulfill this intent. This is the ledger component of a proposal
+for a light client solution.
+
+- Intents that support more advanced kinds of swaps, such as limit or market orders,
+and do not require the entire quantity offered to be swapped at once.
+
+Supporting these sorts of intents require the definition of new types of temporary ledger
+state structures, as well as changes to zone-level constraints.
+We believe that this is a fruitful direction that we can extend in future, and,
+as such, we think we can progress with this design even if we don’t have a full
+picture of where we might want to go with intents, although this is a significant risk.
+However, at this time there is no proposal for a general solution, i.e. one where introducing
+the ability to process each new kind of intent does not require specific ledger
+changes. Instead, the ideal intents-processing solution using validation zones
+would support intents whose zone-level constraints and any temporary data that
+must be recorded are specified _within the zone itself_, providing much more
+flexibility in the kinds of intents that are possible.
+
+It may be that the best way to implement intents on Cardano is entirely different from
+the underspecified transactions and validation zones approach we are taking here.
+In that case such a design might entirely obsolete this one, making it an expensive
+and pointless burden to maintain in future.
+
+
+## Path to Active
+
+### Software Readiness Level
```

Comment:
For consistency with other CIPs (mainly for review in parallel with 100+ others) this section needs to be broken into _Acceptance_ and _Implementation_ ... I guess since it refers to testing functionality then it would be on the Implementation path.

When done sifting material around in this section it will also help for these items to be GitHub formatted tickboxes (`- [ ]`) but I am not going to be pedantic about it especially at this stage.

Reply:
Sounds good, will do that later today!